spec/acceptance/token_expiration.feature

```diff
@@ -1,14 +0,0 @@
-Feature: Approving a proposal from the web application expires the token
-  Scenario: An approver visits the page to approve
```

Comment:
this scenario is covered by https://github.com/18F/C2/blob/master/spec/requests/proposals_spec.rb#L85